### DISCUSSION TITLE
type of presence idea (Listending to TIDAL

### DISCUSSION THREAD

**Discussion by Luc5H** [purpl3F0x/TIDAL-Discord-Rich-Presence-UNOFFICIAL#96] [Ideas]
I barely know what I'm talking about so bear with me here. As far as I'm aware you can now change the type of presence on your application so you can choose between "Playing", "Listening", "Watching" and "Competing". I just saw this as an option with a custom RPC tool that I know of.

![image](https://github.com/purpl3F0x/TIDAL-Discord-Rich-Presence-UNOFFICIAL/assets/77894289/56bb0b02-606e-48bd-aeaf-974ca2484475)

So yeah, if possible, consider changing the type of presence to 'Listening'.

**Comment by purpl3F0x** [comment]
Hello, which application is this ? app already implements that but discord game sdk doesn't work properly.

At least last time i checked

**Reply by Luc5H** [reply]
The application shown in the screenshot is Vencord, it's a custom Discord client that offers a custom RPC plugin. Anyways I found a relevant thread on stack overflow, here: https://stackoverflow.com/questions/73049373/setpresence-activity-type-in-discord-js-v14-can-only-be-set-to-playing

hope this helps

**Comment by purpl3F0x** [comment]
@Luc5H This has been asked before few times, as you can see at line [133](https://github.com/purpl3F0x/TIDAL-Discord-Rich-Presence-UNOFFICIAL/blob/9b8146e7fc735464bb7a0812effd5020854136e8/main.cc#L133) I register the event of `type DiscordActivityType_Listening`.
So it seem's discord sdk is broken

**Reply by purpl3F0x** [reply]
![image](https://github.com/purpl3F0x/TIDAL-Discord-Rich-Presence-UNOFFICIAL/assets/16735391/29301176-37f3-4df4-a357-5c11620280ba)
